.circleci/config.yml

```diff
@@ -565,7 +565,7 @@ jobs:
           create-repo: false
           dockerfile: Dockerfile
           repo: amplify-cli-e2e-base-image-repo-public
-          tag: "latest,${CIRCLE_SHA1}"
+          tag: "node18,${CIRCLE_SHA1}"
```

Comment:
using different tag as new image line won't work should somebody kick-off CCI workflows from older tags/commits/etc.